src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -181,6 +190,7 @@ public StreamingResponseBody streamEvents(
                 client.checkScopes(eventType.getReadScopes());
 
                 // validate parameters
+                final TopicRepository topicRepository = timelineService.getTopicRepository(eventType);
```

Comment:
@adyach ok. I think it's a good idea to deliver it part by part. Let's revisit this when implementing consumer.